src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/SelectionNode.cs

```diff
@@ -84,10 +84,10 @@ private IndexPath IndexPath
 			}
 		}
 
-		// For a genuine tree view, we dont know which node is leaf until we 
-		// actually walk to it, so currently the tree builds up to the leaf. I don't 
-		// create a bunch of leaf node instances - instead i use the same instance m_leafNode to avoid 
-		// an explosion of node objects. However, I'm still creating the m_childrenNodes 
+		// For a genuine tree view, we dont know which node is leaf until we
+		// actually walk to it, so currently the tree builds up to the leaf. I don't
+		// create a bunch of leaf node instances - instead i use the same instance m_leafNode to avoid
+		// an explosion of node objects. However, I'm still creating the m_childrenNodes
```

Comment:
```suggestion
		// For a genuine tree view, we do not know which node is leaf until we
		// actually walk to it, so currently the tree builds up to the leaf. I don't
		// create a bunch of leaf node instances - instead, i use the same instance m_leafNode to avoid
		// an explosion of node objects. However, I'm still creating the m_childrenNodes
```